R/g.analyse.perday.R

```diff
@@ -541,7 +581,7 @@ g.analyse.perday = function(ndays, firstmidnighti, time, nfeatures,
                       p = which(varnum * UnitReScale >= params_phyact[["mvpathreshold"]][mvpai]); rr1[p] = 1
                       getboutout = g.getbout(x = rr1, boutduration = boutduration,
                                              boutcriter = params_phyact[["boutcriter"]],
-                                             closedbout = params_phyact[["closedbout"]],
+                                             closedbout = FALSE, # to match part5 bout calculation
```

Comment:
same comment: closedbout is by default FALSE in params_phyact, so if the user wants this to be TRUE in part 2 then they should be free to change it. So, I think keeping `closedbout = params_phyact[["closedbout"]],` is most intuitive.